spark-icons/src/main/res/res/drawable/spark_icons_shop_2_shop.xml

```diff
@@ -0,0 +1,37 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This particular image is way too big and contains lint warnings

> Very long vector path (6772 characters), which is bad for performance. Considering reducing precision, removing minor details or rasterizing vector.

Do you have the source SVG file? If so we should compress it first (otherwise we should make it a webp). I can do that if you give me the raw file :)

Reply:
Nope I don't but it might be an illustration moved here by mistake 🤔
Let's see where it's used in LBC

Reply:
Okay so I looked it up and this is a FT specific Icon that shouldn't be here.
I'll remove it 👍